README.md

```diff
@@ -52,6 +50,14 @@ jobs:
         app_port: 3000
         additional_tags: "{\"key1\": \"value1\",\"key2\": \"value2\"}"
 
+    - if: ${{ steps.deploy.outputs.vm_url != '' }}
+      run: |
+        echo "## VM Created! :rocket:" >> $GITHUB_STEP_SUMMARY
```

Comment:
Considering how it looks like in the GH Actions
![image](https://user-images.githubusercontent.com/1533818/208996750-99add7f8-0a7b-41dc-a4cb-f72683cebbf0.png)

I think it would be nicer to have a `name:` for the step so something else is printed out in the step title, instead of  "echo"

```suggestion
    - if: ${{ steps.deploy.outputs.vm_url != '' }}
      name: Add Github Summary
      run: |
        echo "## VM Created! :rocket:" >> $GITHUB_STEP_SUMMARY
```

same for the "VM Destroyed one"

Also maybe it makes sense to merge those two steps into one if-then-else block?

Reply:
Added names in the README
There is no option for else.